src/commands/run.ts

```diff
@@ -36,12 +36,13 @@ export const command = new Command()
       config = await Deno.readTextFile(configPath);
     } catch (_e) {
       log.error(`Could not read config ${configPath}`);
-      return {};
+      Deno.exit(1);
     }
+
     const configs = parseConfigYaml(config);
     for (const cfg of configs) {
       const taskMap = await loadTasks(cfg);
-      await runTasks(cfg, taskMap, tasks, options);
+      await runTasks(cfg, taskMap, tasks, configs.length == 1, options);
```

Comment:
instead of making this a calculated config, can you add it as a flag that can propagate via options, i.e. `--fail-missing`?

It would be a good flag to have for CI and builds to ensure that a task you're running is run 100% or causes a failure.